libs/wire-api/src/Wire/API/Routes/Public/Galley/Team.hs

```diff
@@ -26,14 +26,15 @@ import Wire.API.Error.Galley
 import Wire.API.Routes.MultiVerb
 import Wire.API.Routes.Named
 import Wire.API.Routes.Public
+import Wire.API.Routes.Version
 import Wire.API.Team
 import Wire.API.Team.Permission
 
 type TeamAPI =
   Named
     "create-non-binding-team"
     ( Summary "Create a new non binding team"
-        -- FUTUREWORK: deprecated in https://github.com/wireapp/wire-server/pull/2607
+        :> Until 'V4
```

Comment:
maybe still link the PR and/or ticket?  not sure, maybe it's unnecessary noise.

Reply:
I prefer less noise.